sample-apps/donet-sample-app/Controllers/MetricEmitter.cs

```diff
@@ -0,0 +1,137 @@
+using System;
+using System.Collections.Generic;
+using OpenTelemetry;
+using System.Diagnostics.Metrics;
+using OpenTelemetry.Metrics;
+using OpenTelemetry.Instrumentation;
+
+namespace integration_test_app.Controllers
+{
+    public class MetricEmitter
+    {
+        const string DIMENSION_API_NAME = "apiName";
+        const string DIMENSION_STATUS_CODE = "statusCode";
+        
+
+        static string API_COUNTER_METRIC = "totalApiRequests";
+        static string API_LATENCY_METRIC = "latencyTime";
+        static string API_SUM_METRIC = "totalBytesSent";
+        static string API_TOTAL_TIME_METRIC = "timeAlive";
+        static string API_TOTAL_HEAP_SIZE = "heapSize";
+        static string API_TOTAL_THREAD_SIZE = "threadSize";
+        static string API_CPU_USAGE = "cpuUsage";
+    
+
+        Histogram<double> apiLatencyRecorder;
+        Counter<long> totalTimeSentObserver;
+
+        static long apiRequestSent = 0;
+        static long totalBytesSent = 0;
+        static long totalHeapSize  = 0;
+        static long totalThreadSize = 0;
+        static int cpuUsage = 0;
+
+        // The below API name and status code dimensions are currently shared by all metrics observer in
+        // this class.
+        string apiNameValue = "";
+        string statusCodeValue = "";
+
+        
+        public MetricEmitter()
+        {
+            Meter meter = new Meter("aws-otel", "1.0");
+            using var meterProvider = Sdk.CreateMeterProviderBuilder()
+                .AddMeter("aws-otel")
+                .AddOtlpExporter()
```

Comment:
```suggestion
                .AddOtlpExporter()
```
The tracer provider pulls the endpoint from the environment. Do we need to do that here also?

Reply:
It is pulled from the launchSettings.json. In my local testing I didn't have to explicitly mention it here